l10n/en/firefox/browsers/compare/index.ftl

```diff
@@ -30,23 +30,23 @@ compare-index-and-the-winner-is = And the winner is…
 compare-index-since-your-browser-is-your = Since your browser is your gateway to the internet, speed, security, privacy and utility are paramount. In recent years, { -brand-name-google } { -brand-name-chrome } has been the browser of choice for many. But at a time when online ads seem to follow us everywhere and data breaches are a fixture of news headlines, a lot of people are starting to demand more privacy and respect from their browser.
 
 # Obsolete string
 compare-index-the-first-thing-to-point-out = The first thing to point out about portability is that not all browsers run on all operating systems. While { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-brave } and { -brand-name-opera } work on all major systems and are easy to install, { -brand-name-ie } and { -brand-name-safari } only work on { -brand-name-microsoft } and Apple’s own systems. The mobile version of { -brand-name-safari } is pre-installed on Apple’s mobile devices, and most { -brand-name-android } devices come with a pre-installed browser modified by the manufacturer for the device. { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-brave }, { -brand-name-edge } and { -brand-name-opera } can easily be installed and even used side by side.
 
-compare-index-the-first-thing-to-point-out-v2 = The first thing to point out about portability is that not all browsers run on all operating systems. While { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-brave } and { -brand-name-opera } work on all major systems and are easy to install, { -brand-name-safari } only works on { -brand-name-microsoft } and Apple’s own systems. The mobile version of { -brand-name-safari } is pre-installed on Apple’s mobile devices, and most { -brand-name-android } devices come with a pre-installed browser modified by the manufacturer for the device. { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-brave }, { -brand-name-edge } and { -brand-name-opera } can easily be installed and even used side by side.
+compare-index-the-first-thing-to-point-out-v2 = The first thing to point out about portability is that not all browsers run on all operating systems. While { -brand-name-firefox }, Chrome, Edge, Brave and Opera work on all major systems and are easy to install, Safari only works on Microsoft and Apple’s own systems. The mobile version of Safari is pre-installed on Apple’s mobile devices, and most Android devices come with a pre-installed browser modified by the manufacturer for the device. { -brand-name-firefox }, Chrome, Brave, Edge and Opera can easily be installed and even used side by side.
```

Comment:
Unrelated: Safari hasn't been available on Windows in decades? 🤔 
https://en.wikipedia.org/wiki/Safari_(web_browser)#Windows

Reply:
Wow thanks for flagging!
I changed the string to `Safari only works on Apple’s own systems`